tokens/tokens-eth.json

```diff
@@ -16787,6 +16787,31 @@
 			"youtube": ""
 		}
 	},
+	{
+		"symbol": "XPAT",
```

Comment:
Symbol should be PAT based on what I can see from [etherscan](https://etherscan.io/token/0xBB1fA4FdEB3459733bF67EbC6f893003fA976a82)